src/plugins/github/parser.test.js

```diff
@@ -66,38 +66,66 @@ describe("GithubParser", () => {
     });
   });
 
+  function getIssue(n: number): IssueJSON {
+    const issues = exampleRepoData.repository.issues.nodes;
+    const selected = issues.filter((x) => x.number === n);
+    if (selected.length !== 1) {
+      throw new Error(`Failure finding issue #${n}`);
+    }
+    return selected[0];
+  }
+  function getPR(n: number): PullRequestJSON {
+    const pulls = exampleRepoData.repository.pullRequests.nodes;
+    const selected = pulls.filter((x) => x.number === n);
+    if (selected.length !== 1) {
+      throw new Error(`Failure finding PR #${n}`);
+    }
+    return selected[0];
+  }
+  function parseExample(
```

Comment:
Two comments:
  - It's clear enough after reading the function implementation that this parses a specified _subset_ of the example repo, but this wasn't immediately obvious when just reading the signature.
  - I think that the clients are more readable with named arguments: `parseExample({issues: [1], prs: []})`. What do you think?